core/tests/protractor_utils/forms.js

```diff
@@ -348,8 +362,8 @@ var AutocompleteDropdownEditor = function(elem) {
       // Re-close the dropdown.
       await action.sendKeys(
         'Dropdown Element',
-        dropdownElement.element(searchInputLocator),
-        '\n');
+        element(searchInputLocator),
+        protractor.Key.ENTER);
```

Comment:
why is this change missing from webdriverio?

Reply:
removed